File: app/src/main/java/com/erif/contentloader/helper/anim/AnimRes.java
package com.erif.contentloader.helper.anim;

import android.content.Context;
import android.view.animation.AnimationUtils;
import android.view.animation.LayoutAnimationController;

import com.erif.contentloader.R;

public class AnimRes {

    public static LayoutAnimationController fadeIn(Context context) {
        return AnimationUtils.loadLayoutAnimation(context, R.anim.fade_in_layout);
    }

}

File: app/src/main/java/com/erif/contentloaderdemo/helper/anim/AnimRes.kt
package com.erif.contentloaderdemo.helper.anim

import android.content.Context
import android.view.animation.AnimationUtils
import android.view.animation.LayoutAnimationController
import com.erif.contentloaderdemo.R

class AnimRes {

    companion object {

        fun fadeIn(context: Context?): LayoutAnimationController? {
            return AnimationUtils.loadLayoutAnimation(context, R.anim.fade_in_layout)
        }

    }

}